### DISCUSSION TITLE
`xonsh` Why this name for application?

### DISCUSSION THREAD

**Discussion by pktiuk** [xonsh/xonsh#5006] [General]
I am curious why this app is named xonsh.   
Why not something like pybash or pyshell?

TBH I have problems with remembering this name and sometimes when I want to use it I have to google it to recall the launch command.

**Comment by jnoortheen** [comment]
It is named after "conch", another name for shells

**Comment by anki-code** [comment]
Please start your trip around xonsh world from [xonsh-cheatsheet](https://github.com/anki-code/xonsh-cheatsheet/blob/main/README.md):

> What does xonsh mean?
> The "xonsh" word sounds like [conch [kɑːntʃ]](https://www.google.com/search?q=what+is+conch) - a common name of a number of different sea snail or shells (🐚). Thus xonsh is the reference to the shell word that commonly used to name [command shells](https://en.wikipedia.org/wiki/Shell_(computing)).
